examples/display/main/idf_component.yml

```
@@ -2,6 +2,6 @@ description: BSP Display example
 
 dependencies:
   idf: ">=4.4"
-  esp_wrover_kit: ">=1.0.0"
-  # esp32_s2_kaluga_kit: ">=1.0.0"
-  # esp-box: ">=1.0.0"
+  esp_wrover_kit:
+    version: ">=1.0.0"
```

Want be better to limit versions to compatible ones?
```suggestion
    version: "^1.0.0"
```